src/GameBoard.hs

```diff
@@ -1,98 +1,93 @@
 module GameBoard
-  ( winHeight, winWidth, offset
-  , gameWidth, gameHeight
-  , wallWidth
-  , brickWidth, brickHeight
-  , ballRadius
-  , wallUpPos, wallDownPos, wallLeftPos, wallRightPos
-  , Position
-  , Radius
-  , Width
-  , Height
-  , Brick(..)
-  , Game (..)
-  , initialState
-  ) where
+    ( winHeight, winWidth, offset
+    , gameWidth, gameHeight
+    , wallWidth
+    , brickWidth, brickHeight
+    , ballRadius, speedRatio
+    , wallUpPos, wallDownPos, wallLeftPos, wallRightPos
+    , paddleWidth, paddleHeight, paddleStep
+    , Position
+    , Radius
+    , Width
+    , Height
+    , Brick(..)
+    , Game (..)
+    , Paddle (..)
+    , initialState
+    ) where
 
-  import Graphics.Gloss
+import Graphics.Gloss
 
-  -- | position of the window on the screen
-  offset:: Int
-  offset = 100
+-- | position of the window on the screen
+offset:: Int
+offset = 100
 
-  -- | Window width and height
-  winHeight, winWidth :: Int
-  winWidth = floor (gameWidth + wallWidth * 2)
-  winHeight = floor (gameHeight + wallWidth * 2)
+-- | Window width and height
+winHeight, winWidth :: Int
+winWidth = floor (gameWidth + wallWidth * 2)
+winHeight = floor (gameHeight + wallWidth * 2)
 
-  -- | all the widths : brick, game
-  -- wallWidth = thickness
-  gameWidth, wallWidth, brickWidth, paddleWidth :: Width
-  gameWidth = 800
-  wallWidth = 10
-  brickWidth = 100
-  paddleWidth = 100
+-- | all the widths : brick, game, paddle
+-- wallWidth = thickness
+gameWidth, wallWidth, brickWidth, paddleWidth :: Width
+gameWidth = 800
+wallWidth = 10
+brickWidth = 100
+paddleWidth = 100
 
-  -- | speed ration (when the ball hits a brick)
-  speedRatio :: Float
-  speedRatio = 1.5
+-- | speed ration (when the ball hits a brick)
+speedRatio, paddleStep :: Float
+speedRatio = 1.5
+paddleStep = 5
 
-<<<<<<< ef207458608785b7998e8f3d8c39008da8a43327
```

Comment:
And this, my dear, is your reviewer being sorely incompetent at resolving merge conflicts past 7PM !